src/Network/Network/help/Disconnect-AzVirtualNetworkGatewayVpnConnection.md

```diff
@@ -31,12 +31,21 @@ The **Disconnect-AzVirtualNetworkGatewayVpnConnection** cmdlet enables you to di
 
 ## EXAMPLES
 
-### Example
-```
+### Example 1
+```powershell
 PS C:\> Disconnect-AzVirtualNetworkGatewayVpnConnection -ResourceName vnet-gw -ResourceGroupName vnetgwrg -VpnConnectionId @("IKEv2_7e1cfe59-5c7c-4315-a876-b11fbfdfeed4")
 
 ```
 
+### Example 2
+
+Disconnect given connected vpn client connections with a given virtual network gateway. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+Disconnect-AzVirtualNetworkGatewayVpnConnection -ResourceGroupName vnetgwrg -VirtualNetworkGatewayName 'ContosoVirtualNetwork' -VpnConnectionId @('IKEv2_00000000-0000-0000-0000-00000000000000000') -InputObject <PSVirtualNetworkGateway>
```

Comment:
The parameters `ResourceGroupName` and `VirtualNetworkGatewayName` are in a different parameter set than `VpnConnectionId` and `InputObject` and can't be used together because they're mutually exclusive.